### ISSUE TITLE
Could not find io.qameta.allure.gradle.base:allure-base-plugin:2.9.1

### ISSUE THREAD

**Issue by juergenzimmermann** [allure-framework/allure-gradle#67]
#### I'm submitting a ... 
  - [x] bug report
  - [ ] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

#### What is the current behavior?

I made an upgrade of the Allure plugin for Gradle: from `2.8.1` to `2.9.1`. Now I get the following error resp. output:

```
$ gradle test
Starting a Gradle Daemon

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'kunde'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find io.qameta.allure.gradle.base:allure-base-plugin:2.9.1.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/io/qameta/allure/gradle/base/allure-base-plugin/2.9.1/allure-base-plugin-2.9.1.pom
       - https://repo.maven.apache.org/maven2/io/qameta/allure/gradle/base/allure-base-plugin/2.9.1/allure-base-plugin-2.9.1.pom
       - https://repo.spring.io/milestone/io/qameta/allure/gradle/base/allure-base-plugin/2.9.1/allure-base-plugin-2.9.1.pom
     Required by:
         project : > io.qameta.allure:io.qameta.allure.gradle.plugin:2.9.1 > io.qameta.allure.gradle.allure:allure-plugin:2.9.1 > io.qameta.allure.gradle.adapter:allure-adapter-plugin:2.9.1
         project : > io.qameta.allure:io.qameta.allure.gradle.plugin:2.9.1 > io.qameta.allure.gradle.allure:allure-plugin:2.9.1 > io.qameta.allure.gradle.report:allure-report-plugin:2.9.1
```

#### What is the expected behavior?

No errors like the one for `allure-base-plugin`

#### What is the motivation / use case for changing the behavior?

Upgrade should be possible

#### Please tell us about your environment:

| Allure version | 2.15.0 |
| Test framework | JUnit 5.8.1 |
| Allure adaptor | allure-junit5@2.15.0 |
| Generate report using | allure-gradle |

**Comment by eroshenkoam** [comment]
fixed in 2.9.3

**Comment by juergenzimmermann** [comment]
Thank you @eroshenkoam . I can confirm that `2.9.3` works in our projects.
